src/partials/google-tag-manager-noscript.html

```diff
@@ -0,0 +1,3 @@
+<noscript>
+    <iframe src="//www.googletagmanager.com/ns.html?id=GTM-KB96BNB" height="0" width="0" style="display:none;visibility:hidden"></iframe>
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Add a "title" attribute to this <iframe> tag. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3AFrameWithoutTitleCheck)